setup.py

```diff
@@ -25,13 +25,18 @@
     exit(-1)
 
 
+if sys.getwindowsversion().major < 6 or (sys.getwindowsversion().major == 6 and sys.getwindowsversion().minor < 1):
+    print('ERROR: Cannot install on Windows versions less than 7 / Server 2008 R2')
+    exit(-1)
+
+
 if sys.version_info < (3, 4):
     print('ERROR: Python version must be greater or equal to 3.4')
     exit(-1)
 
 
 setup(name='wintrace',
-      version='0.0.1',
+      version='1.0.0',
```

Comment:
I think we should just keep this pre-release for now. 0.1.0?

Reply:
I wanted to follow semantic version for this project, and since there were breaking changes in this version I changed the major version.

Reply:
I will switch it

Reply:
Done.